packages/scripts/CHANGELOG.md

```diff
@@ -1,7 +1,14 @@
 ## 2.1.0 (Unreleased)
 
+### Bug Fix
+
+- Resolves an issue where npm package lint script did not work in Windows environments ([#9321](https://github.com/WordPress/gutenberg/pull/9321)
+
+### Updated Dependencies
+
 - Updated dependencies: `jest`, `npm-package-json-lint` and `read-pkg-up`
```

Comment:
The above `### Updated Dependencies` section isn't documented in the recent [maintaining-changelogs](https://github.com/WordPress/gutenberg/blob/master/CONTRIBUTING.md#maintaining-changelogs) docs update, I added it as the existing line looked out of place per the `### Bug Fix` heading above which of course the deps updates are not _bug fixes_.